src/main/java/exceptions/InvalidCatalogCommandException.java

```diff
@@ -0,0 +1,11 @@
+package exceptions;
+
+public class InvalidCatalogCommandException extends TantouException {
+    public static final String INVALID_CATALOG_COMMAND_MESSAGE = "Invalid catalog command provided! " +
+            "Try \"catalog -a [AUTHOR_NAME] -m [MANGA_NAME]\" instead! " +
+            "And if you want to delete, just add a \"-d\" at the end!";
+
+    public InvalidCatalogCommandException() {
+        super(INVALID_CATALOG_COMMAND_MESSAGE);
+    }
+}
```

Comment:
Well done abstracting out the unique exceptions in the code base, especially since the code base has become increasingly more complex.

Reply:
The guided comments on how to fix the issue are also helpful for users to understand how to use the application.